lms/djangoapps/courseware/views/index.py

```diff
@@ -179,11 +179,11 @@ def _init_new_relic(self):
 
     def _verify_position(self):
         """
-        Verify that the given position is in fact an int.
+        Verify that the given position is in fact an int and at least 1.
```

Comment:
This method doesn't just verify that it has a positive integer.  Expand the docs to reflect the actual behavior.  "Verify that the given position is an integer. If it is not positive, set it to 1.